ducktape/tests/test.py

```diff
@@ -135,6 +136,22 @@ def _escape_pathname(s):
 
 class TestContext(Logger):
     """Wrapper class for state variables needed to properly run a single 'test unit'."""
+    _TEST_ID_DELIMITER = "~~"
```

Comment:
Now I'm recognizing this from the earlier `__repr__` question. Why the magic delimter?